### PR TITLE
emulator-wtf-android-test 1.1.0

### PR DESCRIPTION
**Pull request by tauntz**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by tauntz** [comment]
This is a follow-up to #3296

File: steps/emulator-wtf-android-test/1.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: emulator.wtf API Token
+  token: null
```
Comment (godrei) [inline steps/emulator-wtf-android-test/1.1.0/step.yml:33]: Please add a summary to this input. (All inputs should have a title and summary at least, while the description is optional).

**Review by godrei — CHANGES_REQUESTED** [review]
Hi @tauntz ,

thanks for the changes!

I spotted 2 more issues:
- Missing summary for the `token` input
- In this version of the step the `ew-cli` is included in the step repository and executed by the step. The step refers to its path as: `./ew-cli`. This will not work as the current working directory points to `BITRISE_SOURCE_DIR` (by default the cloned project repository). To overcome this issue we use this: https://github.com/bitrise-steplib/steps-script/blob/master/step.sh#L3

**Comment by godrei** [comment]
Currently the step fails with an error: `chmod: ./ew-cli: No such file or directory`

**Comment by shams-ahmed** [comment]
@tauntz Please let us know when the PR is ready for another review, will close for the time being.

**Pull request closed without merging**